### PR TITLE
Improve product attr docu

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz please review

* in this pull request I want to mention the DefaultProductAttributeFormatter in the product attribute documentation
* DefaultProductAttributeFormatter can also format the labels

### CODE REVIEW

File: commercetools-models/src/main/java/io/sphere/sdk/products/attributes/AttributeTableRow.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    @Nullable
+    String getTranslatedValue();
+
+    /**
+     * Returns the translated and formatted value of the label of the attribute.
+     *
+     * @return label
+     */
+    @Nullable
+    String getTranslatedLabel();
```
Comment (lauraluiz) [inline commercetools-models/src/main/java/io/sphere/sdk/products/attributes/AttributeTableRow.java]: `Translated` is misleading, as even in the comments say it's translated and formatted. Of course `getTranslatedAndFormattedLabel` is both long and ugly. One solution I can think of is to keep just `getLabel` and change the class name to reflect the translated/formatted thing.

File: commercetools-models/src/main/java/io/sphere/sdk/products/attributes/AttributeTable.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     * @return the maximal translated attribute value length or 0 in case of 0 rows
+     */
+    int getMaxTranslatedValueLength();
+
+    /**
+     * Iterates through the rows and finds the longest length of the translated label length.
+     *
+     * @return the maximal translated attribute label length or 0 in case of 0 rows
+     */
+    int getMaxTranslatedLabelLength();
```
Comment (lauraluiz) [inline commercetools-models/src/main/java/io/sphere/sdk/products/attributes/AttributeTable.java]: I think we are crossing a dangerous line here by introducing pure UI-focused elements into the SDK models. Into SDK convenience would be acceptable. Although I cannot really imagine a use case where we need to print the attributes in tables directly from the Java code, except maybe some console printing.

File: commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ProductAttributeDocumentation.java [hunk 3]
```diff
@@ -104,7 +113,21 @@
...
+
+ {@include.example io.sphere.sdk.products.attributes.ProjectNameProductAttributeFormatter}
+
+ <p>The example in action:</p>
+
+ {@include.example io.sphere.sdk.products.attributes.DefaultProductAttributeFormatterDemo}
+
+
+
+ <h3 id="attribute-table-creation-pattern-matching">Creating a table of attributes (old way)</h3>
```
Comment (schleichardt) [inline commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ProductAttributeDocumentation.java]: So should I remove it?

Comment (schleichardt) [inline commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ProductAttributeDocumentation.java]: Hmm, it is very hard to use, I would always go with the new way. I don't see a use case.

**Comment by schleichardt** [comment]
@lauraluiz updated for you

**Comment by schleichardt** [comment]
Yes, it looks a bit weird, but that is the `Pair` or `Tuple2` class of Java SE. :roll_eyes:

**Pull request merged**